### ISSUE TITLE
[🐛 Bug]: Cannot Download ChromeDriver 115 from Existing Download Path

### ISSUE THREAD

**Issue by j-best1** [webdriverio/webdriverio#10805]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md#reporting-new-issues).

### WebdriverIO Version

latest

### Node.js Version

v18.12.1

### Mode

WDIO Testrunner

### Which capabilities are you using?

```typescript
browserName: Chrome
browswerVersion: 115.0.5790.114
platformName: MacBook Pro (Intel)
macOS: Ventura 13.4.1
```

### What happened?

This is my first time creating a github issue, so thank you in advance for any help!

The version of Chrome running on my machine is managed for me. It was recently updated to 115.0.5790.114, so I updated the ChromeDriver version in my app's wdio config (wdio.conf.base.js) file to match. 

After the update to Chrome 115, the tests would not be run because ChromeDriver could not be downloaded. The previous download path seems outdated as of ChromeDriver 115.

The downloads are not available at the previous location: https://chromedriver.storage.googleapis.com
Google has is using another site now: https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing

See https://developer.chrome.com/blog/chrome-for-testing/

Similar, but slightly different from this issue:
https://github.com/webdriverio/webdriverio/issues/10620

### What is your expected behavior?

Download necessary chromedriver version to start and run tests.

### How to reproduce the bug.

To reproduce the bug:
1. Update Chrome to 115+
2. Update wdio configuration to expect to use ChromeDriver 115+.
3. Attempt to run tests.

### Relevant log output

```typescript
Error in "getDownloadStream". Could not download https://chromedriver.storage.googleapis.com/115.0.5790.102/chromedriver_mac64.zip
See more details below:
404 https://chromedriver.storage.googleapis.com/115.0.5790.102/chromedriver_mac64.zip
Response code 404 (Not Found)
~/node_modules/selenium-standalone/lib/install.js:292
          throw new Error('Could not download ' + downloadUrl);
          ^

Error: Could not download https://chromedriver.storage.googleapis.com/115.0.5790.102/chromedriver_mac64.zip
    at Request.<anonymous> (~/node_modules/selenium-standalone/lib/install.js:292:17)
    at Object.onceWrapper (node:events:628:26)
    at Request.emit (node:events:513:28)
    at Request.emit (node:domain:489:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at processTicksAndRejections (node:internal/process/task_queues:82:21)

Node.js v18.12.1
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by j-best1** [comment]
I realized after creating this issue that the underlying problem is actually in selenium-standalone.
An issue was created there a couple days ago: https://github.com/webdriverio/selenium-standalone/issues/810

**Comment by christian-bromann** [comment]
Yes, this issue is a duplicate of that. Also we are working on bringing driver management into WebdriverIO in #10767 so this will be fixed once that is merged.

**Comment by j-best1** [comment]
Thank you so much! I appreciate the response and the work you are doing to fix the issue.
